lib/puppet/ssl/certificate_request.rb

```diff
@@ -267,7 +267,17 @@ def extension_request_attribute(options)
 
     if options[:dns_alt_names]
       names = options[:dns_alt_names].split(/\s*,\s*/).map(&:strip) + [name]
-      names = names.sort.uniq.map {|name| "DNS:#{name}" }.join(", ")
+
+      names = names.sort.map do |name|
```

Comment:
It's a good thing to sort, but with the inputs now being (potentially) prefixed it might make sense to sort after we ensure they're prefixed.